airbyte-webapp/src/packages/cloud/views/auth/SignupPage/components/SignupForm.tsx

```diff
@@ -232,6 +234,12 @@ export const SignupForm: React.FC = () => {
             </RowFieldItem>
           )}
 
+          {/* exp-select-source-signup */}
+          {showSourceSelector && (
+            <FieldItem>
```

Comment:
:broom: Once not an experiment anymore, this value should properly be tracked via Formik state and handled only in the `onSubmit` of that form to be stored in state (and likely also use another mechanism than `localStorage` to store it and read it out later, but instead do the redirect directly in the `onSubmit`.